docs/admin/install/steps/install-after.rst

```diff
@@ -13,7 +13,7 @@ Congratulations, your Weblate server is now running and you can start using it.
 Adding translation
 ++++++++++++++++++
 
-#. Open the admin interface (``http://localhost:8000/create/project/``) and create the project you
+#. Open the admin interface (``http://localhost:8000/create/project``) and create the project you
```

Comment:
The URL has a trailing slash...

Reply:
/create/project redirects to /create/project/ anyway

Reply:
Yes, it redirects, but why to change the working URL to one which redirects?